### DISCUSSION TITLE
Unable to import dependencies from Azure

### DISCUSSION THREAD

**Discussion by mflova** [python-poetry/poetry#8111] [Q&A] [answered]
I am setting up a `Python` project with a few dependencies. Some of them are public (PyPi) and some others are `Python` git-projects set up with `Poetry`. After reading the [documentation](https://python-poetry.org/docs/dependency-specification/#git-dependencies), it seems that you can modify `pyproject.toml` to indicate these git paths.

```
[tool.poetry.dependencies]
python = "^3.9"
my_repo = {git = "https://{...}@dev.azure.com/{...}/my-repo", branch="develop"}
```

I have tried with a public github repository and it is working well. However, when it comes to referencing a repository in `Azure`, I get the following error after calling `poetry install`:

```shell
  ò Installing my_repo (0.1.0 5662892)

  BuildSystemTableValidationError

  Failed to validate `build-system` in pyproject.toml: Unknown properties: paths

  at ~\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\build\__init__.py:231 in _parse_build_system_table
      227|         raise BuildSystemTableValidationError('`backend-path` must be an array of strings')
      228| 
      229|     unknown_props = build_system_table.keys() - {'requires', 'build-backend', 'backend-path'}
      230|     if unknown_props:
    > 231|         raise BuildSystemTableValidationError(f'Unknown properties: {", ".join(unknown_props)}')
      232| 
      233|     return build_system_table
      234| 
      235| 
```

More information about the issue:
- Using Windows10 (I do not own full permissions)
- I have full access to this azure repo. I can clone it within the terminal, so I guess it is not about permissions over the remote repository
- In the `git` path I also tried with the URL, SSH and HTTPS, but nothing worked
- This repo I am importing (`my_repo`), has its corresponding `pyproject.toml` and I can install it in my station with `poetry install`. It is a really basic one so it is not complex from the point of view of settings, configurations or setup
- If this repository is locally in my PC, it works if I add the dependency as:

```toml
[tool.poetry.dependencies]
python = "^3.9"
my_repo = {path = "<PATH_TO_MY_REPO>"}
```

I already tried everything. So I would be really helpful if somebody can give me some hints about what's going on :)

**Comment by radoering** [comment, marked as answer]
According to the error message, validation of your `pyproject.toml` fails because you have a `paths` key under `build-system`. What does this section look like in your `pyproject.toml`?

**Reply by mflova** [reply]
Thanks for taking your time to read it! I have just found the issue.

The one located in my local path was not the same as the one in the git repository (although I thought they were). I was checking the wrong revision. The one that `poetry` was referencing had a wrong line with `paths` keyword in `pyproject.toml`. I will remove that line.
